environment.devenv.yml

```diff
@@ -18,14 +18,14 @@ dependencies:
 
   {% if TEST_QMXGRAPH %}
   - cherrypy
-  - hypothesis ==3.11.0
+  - hypothesis =3.11
   - pytest-mock
   - pytest-qt
   - pytest-rerunfailures
   - pytest-selenium >=1,<2
   - pytest-timeout
-  {% if sys.platform != 'win32' %}
-  - pytest-xvfb
-  {% endif %}
-
+  - typing-extensions <4.2.0  # [PYTHON_VERSION=="3.6"]
```

Comment:
I think we always default `PYTHON_VERSION` to "3.6" given we never explicitly set it.